### DISCUSSION TITLE
Lets invite token will have tunable expiration time!

### DISCUSSION THREAD

**Discussion by belonesox** [sagemathinc/cocalc#6483] [Ideas]
Now invite tokens have fixed two weeks expiration time.

For my experience, for most my projects/courses I have to repeatedly refresh token, it is so boooring.

Lets invite token will have tunable expiration time!
May be discrete, 2w, 1m, 3m, 1y, ∞ … (to simplify UI).

**Comment by williamstein** [comment]
This is a good idea. I would be happy if you create an issue for this.
